### ISSUE TITLE
Run inference on new WSIs

### ISSUE THREAD

**Issue by irina-zys** [bupt-ai-cz/glomeruli-instance-segmentation#2]
hi, thanks a lot for sharing the model :)

I wonder if you can give some brief instruction on running inferences on new images using your model? Thanks!

**Comment by AmitSharma1127** [comment]
Yes, that would be helpful. I am already having a hard time even setting up the environment via mmdetection. The instructions could be more precise and clear :)
